doc/select.md

```diff
@@ -0,0 +1,204 @@
+# Select examples
+
+## Filtering
+
+The second parameter passed to ``crud.select`` is an array of conditions. Below are examples of filtering data using these conditions. 
+
+### Getting full space
+
+To get a full space without filtering, you need to pass ``nil`` as a condition.
+
+**Example:**
+
+```lua
+crud.select('customers', nil)
```

Comment:
This is a bad approach to select all space contents. I think we should show user that requests without conditions should be performed only with `first` specified.

You can write smth like:

Let's check `customers` space contents to make other examples more clear. Just select first 10 values without conditions.